reference/checkouts.sf.yml

```diff
@@ -1134,7 +1147,42 @@ components:
           type: number
           description: The discounted amount applied within a given context.
           format: double
-    Customer: 
+    CheckoutFee:
+      title: Checkout Fee
+      type: object
+      properties:
+        id:
+          type: string
+          format: uuid
+          description: The fee ID.
+          example: 497f6eca-6276-4993-bfeb-53cbbbba6f08
+        type:
+          type: string
+          description: The type of the fee
+          enum:
+            - custom_fee
+        name:
+          type: string
+          description: Name of the fee.
+          example: ROUTEINS
+        displayName:
+          type: string
+          description: Display name of the fee targeting customers/shoppers
+          example: Package Protection Insurance
+        cost:
+          type: number
+          description: Cost of the fee (include or exclude tax dependent on tax settings, as same as shipping cost)
+          example: 10.0
+        source:
+          type: string
+          description: The source of the request
+          example: Route
```

Comment:
LGTM, I'm just not sure if we should be using Route as example or give a random one instead to avoid the advertisment 😄

Reply:
yes, good point. let me change it lol